host/src/connection.rs

```diff
@@ -31,6 +31,20 @@ pub struct ConnectParams {
     pub supervision_timeout: Duration,
 }
 
+/// An event
+#[derive(Clone)]
+pub enum ConnectionEvent<'d> {
+    /// Connection disconnected.
+    Disconnected {
+        /// The reason (status code) for the disconnect.
+        reason: Status,
+    },
+    #[allow(unused, missing_docs)]
+    /// GATT write event.
+    // TODO: Might need to wrap this in a type that holds a Pdu<'d> or a resources from a pool
+    GattWrite { data: &'d [u8] },
```

Comment:
Yeah, I can remove it. I had originally put it in there so that I could wire the lifetimes of the ConnectionEvent correctly, since that would be required for events that reference Pdu's that were allocated.

Reply:
Maybe it can stay for now, since it will probably make life easier for anyone making the change to provide gatt write events?

Reply:
Yep, perhaps just a docstring so an API user knows not to expect any data from it yet?